IuliiaMazhara/IuliiaMazhara
# Hm 5 i

Test structure requirements:
1.	Browser – Chrome
2.	Window – maximized
3.	Web Elements locators should not be autogenerated to browsers plugins
Priority: 
•	DOM selector
•	CSS selector
•	XPATH selector
4.	All code should be separated by functional methods
5.	Code must conform to the [Java Code Convention](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf)
6.	Tests should be run through the Test NG suite xml file.
7.	Should be created maven profiles for hw5
8.	Checkstyle plugin should not produce errors
9.	All tests must launch through the maven command

File placement requirements:
1.	Classes should be placed into appropriate package and sub-packages
2.	Two test cases > Two Cucumber Feature files
3.	Test methods should have a name in accordance with checking functionality
4.	Translate the test case to BDD Gherkin style
5.	Place finished Gherkin test into hw6 package with corresponding extension and scope (main/test/resources)
6.	Make associations Gherkin annotations step design patterns
